<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Программа для создания проектов по встраиваемым системам</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditProjectName"/>
    </item>
    <item row="2" column="2">
     <widget class="QRadioButtonTree" name="radioTree">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Микроконтроллеры</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButtonCreate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Путь</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButtonFileDialog">
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEditPath">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Имя проекта</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="labelProgress">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Загрузить список</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Локально</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Удаленно</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Масштаб списка</string>
     </property>
     <addaction name="tree_font_increase"/>
     <addaction name="tree_font_decrease"/>
    </widget>
    <addaction name="menu_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="tree_font_increase">
   <property name="text">
    <string>Увеличить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="tree_font_decrease">
   <property name="text">
    <string>Уменьшить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRadioButtonTree</class>
   <extends>QTreeWidget</extends>
   <header>qradiobuttontree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonFileDialog</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>editPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>762</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCreate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>createProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditProjectName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>checkCyrillic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>downloadConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cloneConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menubar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>performAction(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>cyrillicPathWarning()</signal>
  <slot>editPath()</slot>
  <slot>createProject()</slot>
  <slot>checkCyrillic()</slot>
  <slot>downloadConfig()</slot>
  <slot>cloneConfig()</slot>
  <slot>performAction(QAction*)</slot>
 </slots>
</ui>
